### ISSUE TITLE
need to revert NYU-CI/RCPublications#44

### ISSUE THREAD

**Issue by ceteri** [NYU-CI/RCPublications#46]
@JasonZhangzy1757 @srand525  there's an error in commit #44 which shows up downstream in the graph -- the dataset list is corrupted:

```
"title": "Citizen science datasets reveal drivers of spatial and temporal variation for anthropogenic litter on Great Lakes beaches",
    "datasets": [
      "dataset-617\u00a3\u00acdataset-632"
    ],
```

**Comment by JasonZhangzy1757** [comment]
Sorry it's my fault, and I'm dealing with it right now.

**Comment by srand525** [comment]
fixed by #48

**Comment by ceteri** [comment]
Many thanks @JasonZhangzy1757  --

...and I always try to blame the code, not a person :)
